api/DHSC.FingertipsNext.Modules.Core.Repository/Dimensions/Models/AgeDimension.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DHSC.FingertipsNext.Modules.Repository.Dimensions.Models
+{
+    [Serializable]
+    public sealed class AgeDimension
```

Comment:
Can we avoid sealing classes unless absolutely necessary please - sealed by exception rather than by rule.